conf.yaml

```diff
@@ -640,8 +640,8 @@ contents:
             prefix:     en/cloud
             tags:       Cloud/Reference
             subject:    Elastic Cloud
-            current:    ms-57
-            branches:   [ {'ms-57': latest} ]
+            current:    ms-60
```

Comment:
This is the version for ESS. I don't think you want to change this (it might also conflict with #2137 which is updating this to `ms-58`)

Reply:
I think you want to update line 751 instead:

```
branches: [ {'ms-60': 2.10}, {'ms-53': 2.9}, {'ms-49': 2.8}, {'ms-47': 2.7}, {'release-ms-41': 2.6}, 2.5, 2.4, 2.3, 2.2, 2.1, 2.0, 1.1, 1.0 ]
```

After building, I'll have to confirm that "2.10" is still considered "newer" than "2.9". Around the time of the 7.10 Elastic release, I had to update the logic so it wasn't seen as (numerically) "equal" to "7.1".

Reply:
Thanks @gtback ! I have to admit I have no clue how this config works and what it does, I've been just blindly following the instructions in the release guide...
One thing about ECE 2.10 is that **it's based off ms-57** milestone branch. Please have a look at my changes now 🙂

Reply:
Looks to me this is still the wrong section. It changes ESS docs not ECE. ECE is at line 751.

Reply:
Thank you, I think this looks better now with the new commit